src/Momento.Sdk/Internal/DataGrpcManager.cs

```diff
@@ -102,12 +101,6 @@ public async Task<_DictionaryFetchResponse> DictionaryFetchAsync(_DictionaryFetc
         return await wrapped.ResponseAsync;
     }
 
-    public async Task<_DictionaryDeleteResponse> DictionaryDeleteAsync(_DictionaryDeleteRequest request, CallOptions callOptions)
```

Comment:
do we not have one of these for lists or sets?

Reply:
We do. The RPCs that delete a whole list or whole set have multiple uses, so we need to keep those. Because the dictionary delete RPC only has that one use, it's the only one we can delete outright.